src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -369,3 +372,16 @@ def _invalid_sku_update():
 
 def _invalid_sku_downgrade():
     raise CLIError("Managed registries could not be downgraded to Classic SKU.")
+
+
+def validate_and_serialize_build_arguments(
```

Comment:
nit: not much serialization going on here, maybe just validate_and_append or something, with a NB to indicate whether or not the original structure will remain immutable or not

Reply:
Done.